scheduler.go

```diff
@@ -209,6 +210,25 @@ func (s *Scheduler) removeByCondition(shouldRemove func(*Job) bool) {
 	}
 }
 
+// Remove First found Job by Tag
+func (s *Scheduler) RemoveJobByTag(tag string) {
```

Comment:
Return `error`?
If the job was not found (`jobindex` < 0), the client using this will never know

Reply:
oh i see, so it' will not confuse the client who using this lib. thank you